### DISCUSSION TITLE
Print constraint values using linear solver model builder

### DISCUSSION THREAD

**Discussion by arnabadas** [google/or-tools#3697] [Feature requests] [answered]
**What language and solver does this apply to?**
Python, Linear Solver Model Builder (SCIP/SAT, should be applicable to other backends as well)

**Describe the problem you are trying to solve.**
A simple linear integer programming problem

**Describe the solution you'd like**
Get value of the constraint from the solver API without computing it again

**Describe alternatives you've considered**
In case of the basic linear_solver api, I use `ComputeConstraintActivities` to print the constraint values. But there is no such option in the modelsolver API of the linear solver model builder.

**Comment by lperron** [comment, marked as answer]
Implemented on main.

python: solver.activity(ct)
java: solver.getActivity(ct)
